Lib/_pyrepl/unix_console.py

```diff
@@ -525,6 +528,14 @@ def clear(self):
         self.__posxy = 0, 0
         self.screen = []
 
+    def __enable_bracketed_paste(self) -> None:
+        os.write(self.output_fd, b"\x1b[?2004h")
+        self.flushoutput()
```

Comment:
Not sure but I think that this is not necessary since you're writing with `os.write` directly.